apps/console.py

```diff
@@ -271,7 +272,8 @@ async def run_async(self, device_config, transport):
             if device_config:
                 self.device = Device.from_config_file_with_hci(device_config, hci_source, hci_sink)
             else:
-                self.device = Device.with_hci('Bumble', 'F0:F1:F2:F3:F4:F5', hci_source, hci_sink)
+                random_address = ':'.join(map(lambda c: f"{c:02X}", random.sample(range(255), 6)))
```

Comment:
The two ms bits should be 0b11 for static random addresses.

Reply:
Woops! Fixed.